src/wasm.h

```diff
@@ -647,7 +647,7 @@ enum StringEqOp {
 
 class Expression {
 public:
-  enum Id {
+  enum Id : int8_t {
```

Comment:
I think you want uint8_t here if starting at 0, otherwise you can only get 128 values.